test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/SUT/CollectionSerializerCustomersController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Linq;
+using Microsoft.AspNet.OData.Test.Abstraction;
+using Microsoft.AspNet.OData.Query;
+
+namespace Microsoft.AspNet.OData.Test.Formatter.Class
+{
+    public class CollectionSerializerCustomersController : TestODataController
+    {
+        public ITestActionResult Get(ODataQueryOptions<CollectionSerializerCustomer> options)
+        {
+            IQueryable<CollectionSerializerCustomer> customers = new[]
+            {
+                    new CollectionSerializerCustomer{ID = 1, Name = "Name 1"},
+                    new CollectionSerializerCustomer{ID = 2, Name = "Name 2"},
+                    new CollectionSerializerCustomer{ID = 3, Name = "Name 3"},
+                }.AsQueryable();
+
+            IQueryable<IEdmEntityObject> appliedCustomers = options.ApplyTo(customers) as IQueryable<IEdmEntityObject>;
+
+            return Ok(appliedCustomers);
+        }
+    }
+}
+
+namespace Microsoft.AspNet.OData.Test.Formatter.Interface
```

Comment:
Split this to two namespaces for consistency.

Reply:
not sure about what you are asking here? been awhile since looking at this code.

Reply:
I think you are asking about the sut class version in a class namespace and the interface implementation in an interface namespace. is that correct?

Reply:
it might be best to split into two separate files to simplify reading the code base.

Reply:
I need more eyes on this, since I have been unable to resolve the issue with the unit tests not running. @marabooy, I have sent you a collaboration request. I have hit a road block and the work crews appear to be on a permanent coffee break.